src/datarecord.cpp

```diff
@@ -544,9 +558,9 @@ void datarecord::schedule(std::vector<rec_ptr>& thisi, double maxtime,
   }
   
   if(this->int_infusion()) {
-    thisi.reserve(thisi.size() + Addl);  
+    thisi.reserve(thisi.size() + n_dose);  
   } else {
-    thisi.reserve(thisi.size() + Addl); 
+    thisi.reserve(thisi.size() + n_dose); 
```

Comment:
Not introduced by this PR, but it seems a bit confusing to me to have an `if` and `else` branch that are identical.  It's been that way since 638927ba (2019-09-27).  Is there any point in keeping the condition around?

Reply:
Thanks, @kyleam; yes that should be removed; we used to create two records for additional infusions (start record and end record) so it was `2*Addl` when it was an infusion. Now, we create the stop record when the infusion is actually started.